generate-sources/new-plugin/src/main/kotlin/io/arrowkt/example/AddAnnotatedLogging.kt

```diff
@@ -0,0 +1,52 @@
+package io.arrowkt.example
+
+import arrow.meta.CliPlugin
+import arrow.meta.Meta
+import arrow.meta.invoke
+import org.jetbrains.kotlin.backend.common.extensions.IrPluginContext
+import org.jetbrains.kotlin.backend.common.lower.DeclarationIrBuilder
+import org.jetbrains.kotlin.ir.builders.irBlockBody
+import org.jetbrains.kotlin.ir.builders.irCall
+import org.jetbrains.kotlin.ir.builders.irGetObject
+import org.jetbrains.kotlin.ir.declarations.IrFunction
+import org.jetbrains.kotlin.ir.expressions.IrBody
+import org.jetbrains.kotlin.ir.util.getSimpleFunction
+import org.jetbrains.kotlin.ir.util.hasAnnotation
+import org.jetbrains.kotlin.ir.util.statements
+import org.jetbrains.kotlin.name.FqName
+
+val Meta.addAnnotatedLogging: CliPlugin
+    get() = "Add Logging To Annotated Functions" {
+        meta(
+            irFunction { declaration ->
+
+                // FIXME #58: NoClassDefFound when trying to access via MetaLog::class.qualifiedName
```

Comment:
you may have to do `::class.java.canonicalName` or something like that with java reflection

Reply:
Not working also. Well, creating `FqName` with string seems to be OK. Removed `FIXME`